src/scenarios/shared/runner.py

```diff
@@ -361,14 +361,21 @@ def run(self):
             ]
             animator_duration_scale_cmd = RunCommand(cmdline, verbose=True)
             animator_duration_scale_cmd.run()
-            getLogger().info(f"Retrieved values window {window_animation_scale_cmd.stdout.strip()}, transition {transition_animation_scale_cmd.stdout.strip()}, animator {animator_duration_scale_cmd.stdout.strip()}")
+            cmdline = [
+                adb.stdout.strip(),
+                'shell', 'settings', 'get', 'system', 'screen_off_timeout'
+            ]
+            screen_off_timeout_cmd = RunCommand(cmdline, verbose=True)
+            screen_off_timeout_cmd.run()
+            getLogger().info(f"Retrieved values window {window_animation_scale_cmd.stdout.strip()}, transition {transition_animation_scale_cmd.stdout.strip()}, animator {animator_duration_scale_cmd.stdout.strip()}, screen timeout {screen_off_timeout_cmd.stdout.strip()}")
 
             # Make sure animations are set to 1 or disabled
-            getLogger().info("Setting animation values")
+            getLogger().info("Setting needed values")
             if(self.animationsdisabled):
                 animationValue = 0
             else:
                 animationValue = 1
+            minimumTimeoutValue = 120000 # Set the timeout minimum to 2 minutes
```

Comment:
[minor]
Slightly different (comment) style that survives future code edits better
`minimumTimeoutValue = 2 * 60 * 1000 # milliseconds`

or if python 3.6+ is guaranteed
`minimumTimeoutValue = 2 * 60_000 # milliseconds`

or 120 is a recognizable enough number to even justify
`minimumTimeoutValue = 120_000 # milliseconds`